Compilation errors of <CGAL/Polygon_mesh_processing/measure.h>

## Issue Details

That is [the issue reported today on Nabble](http://cgal-discuss.949826.n4.nabble.com/Error-on-master-branch-build-in-measure-h-td4663101.html).

There is a compilation error in `<CGAL/Polygon_mesh_processing/measure.h>`:
```
In file included from /home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp:1:0:
/home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/../../include/CGAL/Polygon_mesh_processing/measure.h: In instantiation of ‘typename CGAL::GetGeomTraits<TriangleMesh, CGAL::cgal_bgl_named_params<T, Tag, Base> >::type::FT CGAL::Polygon_mesh_processing::volume(const TriangleMesh&, const CGAL::cgal_bgl_named_params<T, Tag, Base>&) [with TriangleMesh = CGAL::Surface_mesh<CGAL::Point_3<CGAL::Epick> >; T = bool; Tag = CGAL::internal_np::all_default_t; Base = boost::no_property; typename CGAL::GetGeomTraits<TriangleMesh, CGAL::cgal_bgl_named_params<T, Tag, Base> >::type::FT = double]’:
/home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/../../include/CGAL/Polygon_mesh_processing/measure.h:533:18:   required from ‘typename CGAL::Kernel_traits<typename CGAL::property_map_value<PolygonMesh, boost::vertex_point_t>::type>::Kernel::FT CGAL::Polygon_mesh_processing::volume(const TriangleMesh&) [with TriangleMesh = CGAL::Surface_mesh<CGAL::Point_3<CGAL::Epick> >; typename CGAL::Kernel_traits<typename CGAL::property_map_value<PolygonMesh, boost::vertex_point_t>::type>::Kernel::FT = double]’
/home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp:168:35:   required from ‘void test_closed_surface_mesh(const char*) [with Surface_mesh = CGAL::Surface_mesh<CGAL::Point_3<CGAL::Epick> >; K = CGAL::Epick]’
/home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp:182:89:   required from here
/home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/../../include/CGAL/Polygon_mesh_processing/measure.h:522:10: error: ‘exact’ was not declared in this scope, and no declarations were found by argument-dependent lookup at the point of instantiation [-fpermissive]
     exact(volume);
     ~~~~~^~~~~~~~
In file included from /home/lrineau/Git/cgal/Number_types/include/CGAL/Lazy_exact_nt.h:47:0,
                 from /home/lrineau/Git/cgal/Number_types/include/CGAL/internal/Exact_type_selector.h:34,
                 from /home/lrineau/Git/cgal/Filtered_kernel/include/CGAL/Filtered_kernel.h:36,
                 from /home/lrineau/Git/cgal/Kernel_23/include/CGAL/Exact_predicates_inexact_constructions_kernel.h:30,
                 from /home/lrineau/Git/cgal/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp:2:
/home/lrineau/Git/cgal/Filtered_kernel/include/CGAL/Lazy.h:99:20: note: ‘const CGAL::Bbox_3& CGAL::exact(const CGAL::Bbox_3&)’ declared here, later in the translation unit
   inline const T & exact (const T& d) { return d; } \
                    ^
/home/lrineau/Git/cgal/Filtered_kernel/include/CGAL/Lazy.h:99:20: note: in definition of macro ‘CGAL_LAZY_FORWARD’
   inline const T & exact (const T& d) { return d; } \
                    ^~~~~
```

## Source Code

`Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp` with this patch:

```patch
diff --git a/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp b/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp
index 3565c07..a63da16 100644
--- a/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp
+++ b/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp
@@ -1,10 +1,10 @@
+#include <CGAL/Polygon_mesh_processing/measure.h>
 #include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
 #include <CGAL/Exact_predicates_exact_constructions_kernel.h>
 
 #include <CGAL/Polyhedron_3.h>
 #include <CGAL/Surface_mesh.h>
 
-#include <CGAL/Polygon_mesh_processing/measure.h>
 #include <CGAL/Polygon_mesh_processing/bbox.h>
 
 #include <CGAL/Bbox_3.h>
```

## Environment

* CGAL version: 4.10, maybe even before

--- a/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/measure.h
+++ b/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/measure.h
@@ -34,6 +34,10 @@
 #include <CGAL/squared_distance_3.h>
 #include <CGAL/Kernel/global_functions_3.h>
 
+#include <CGAL/Lazy.h> // needed for CGAL::exact(FT)/CGAL::exact(Lazy_exact_nt<T>)
+
+#include <utility>
+
 #ifdef DOXYGEN_RUNNING
 #define CGAL_PMP_NP_TEMPLATE_PARAMETERS NamedParameters
 #define CGAL_PMP_NP_CLASS NamedParameters

--- a/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp
+++ b/Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp
@@ -1,10 +1,10 @@
+#include <CGAL/Polygon_mesh_processing/measure.h>
 #include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
 #include <CGAL/Exact_predicates_exact_constructions_kernel.h>
 
 #include <CGAL/Polyhedron_3.h>
 #include <CGAL/Surface_mesh.h>
 
-#include <CGAL/Polygon_mesh_processing/measure.h>
 #include <CGAL/Polygon_mesh_processing/bbox.h>
 
 #include <CGAL/Bbox_3.h>

